halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/OrcaProfileFactory.java

```diff
@@ -73,6 +79,20 @@ protected void setProfile(
     profile.appendContents("pipelineTemplates.enabled: " + pipelineTemplates);
     // For backward compatibility
     profile.appendContents("pipelineTemplate.enabled: " + pipelineTemplates);
+
+    final List<Plugin> plugins = deploymentConfiguration.getPlugins().getPlugins();
```

Comment:
Why are we directly appending to the `orca.yml` config file?  Aren't we writing out a separate `plugins.yml` file that will contain the plugin config and that each service supporting plugins will look at?

Reply:
We are writing out a `plugins.yml` file, it is for use prior to spring context initialization and contains the plugin names and the locations of jars to download. `orca.yml` will contain the configurations for the plugins, eg `url` for a service.